src/i18n/pt-br.js

```diff
@@ -0,0 +1,445 @@
+            ok: "SIM"
+        },
+        password: {
+            title: "Palavra-passe",
+            message: "Introduza a sua palavra-passe para continuar"
+        },
+        registerServiceNode: {
+            title: "Registar um nódulo de serviço",
+            message: "Pretende registar um nódulo de serviço?",
+            ok: "REGISTAR"
+        },
+        rescan: {
+            title: "Re-examinar carteira",
+            message: "Aviso: Algumas informações acerca de transações passadas serão perdidas,\ntal como o endereço dos recipientes que serão perdidos.",
+            ok: "RE-EXAMINAR"
+        },
+        restart: {
+            title: "Reiniciar",
+            message: "Alterações requerem um reiniciar. Quer reiniciar agora?",
+            ok: "Reiniciar"
+        },
+        showPrivateKeys: {
+            title: "Mostrar chaves privadas",
+            message: "Deseja ver as suas chaves privadas?",
+            ok: "MOSTRAR"
+        },
+        stake: {
+            title: "Participação num nódulo de serviço",
+            message: "Deseja participar num nódulo de serviço?",
+            ok: "Participação"
+        },
+        sweepAll: {
+            title: "Varrer Tudo",
+            message: "Quer reamente varrer tudo?",
+            ok: "VARRER TUDO"
+        },
+        switchWallet: {
+            title: "Mudar de carteira",
+            closeMessage: "Tem a certeza que deseja fechar a carteira actual?",
+            restartMessage: "A carteira RPC está correntemente em sincronização. \nISe deseja mudar de carteira terá de reiniciar a aplicação. \nPerderá o progresso da sincronização e terá de re-examinar o blockchain."
+        },
+        transactionDetails: {
+            title: "Detalhes das transações",
+            ok: "FECHAR"
+        },
+        transfer: {
+            title: "Transferir",
+            message: "Deseja enviar a transação?",
+            ok: "ENVIAR"
+        },
+        unlockConfirm: {
+            title: "Confirmar desbloqueamento",
+            ok: "DESBLOQUEAR"
+        },
+        unlockServiceNode: {
+            title: "Desbloquear nódulo de serviço",
+            confirmTitle: "Confirmar desbloqueio",
+            message: "Deseja desbloquear o nódulo de serviço?",
+            ok: "DESBLOQUEAR"
+        }
+    },
+    fieldLabels: {
+        // Field labels are also all uppercased
+        address: "ENDEREÇO",
+        amount: "QUANTIDADE",
+        confirmPassword: "CONFIRMAR PALAVRA-PASSE",
+        daemonLogLevel: "NÍVEL DE LOG PARA O SERVIÇO",
+        daemonP2pPort: "PORTA P2P DO SERVIÇO",
+        daemonZMQPort: "PORTA ZMQ DO SERVIÇO",
+        dataStoragePath: "DIRECTÓRIO DE ARMAZENAMENTO DOS DADOS",
+        filterTransactionType: "FILTRAR POR TIPO DE TRANSAÇÃO",
+        filterTxId: "FILTRAR POR TXID",
+        internalWalletPort: "PORTA INTERNA DA CARTEIRA",
+        keyImages: {
+            exportDirectory: "DIRECTORIA DE EXPORTAÇÃO DAS CHAVES DE IMÁGEM",
```

Comment:
exportDirectory: "DIRECTORIA DE EXPORTAÇÃO DAS CHAVES DE IMAGEM"

Reply:
Yeah, I'm pre Acordo Ortográfico schooled, so it's automatic. Will fix